client/tests/integration/components/collection-item-test.js

```diff
@@ -1,25 +0,0 @@
-import { moduleForComponent, test } from 'ember-qunit';
-import hbs from 'htmlbars-inline-precompile';
-
-moduleForComponent('collection-item', 'Integration | Component | collection item', {
```

Comment:
There are still components for `collection-item`, `collection-nav`, and `item-detail-page` so should these test files be deleted?